drupal/sites/default/boinc/modules/boinccore/boinccore.admin.inc

```diff
@@ -227,3 +227,41 @@ function boinccore_admin_default_content_generate($pages = array()) {
   menu_cache_clear_all();
   return $pages_generated;
 }
+
+/**
+ * BOINC form to set database options
+ **/
+function boinccore_admin_dbopts(&$form_state) {
+  $form = array();
+  $default = array(
+    'database_encoding' => variable_get('boinc_database_encoding', 'utf-8')
```

Comment:
The default encoding for the BOINC DB should be `latin1`. That's the mysql default and should still be the encoding used for most of the existing projects. It's also the encoding used for new projects if they didn't switch to UTF8 manually.